scrapers/MatureNL.yml

```diff
@@ -0,0 +1,43 @@
+name: "Mature.nl"
+sceneByURL: &byUrl
+  - action: scrapeXPath
+    url:
+      - mature.nl/
+    scraper: sceneScraper
+galleryByURL: *byUrl
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &title //div[@class="box"]/h1/text()
+      Details: &details //meta[@name="description"]/@content
+      Tags: &tags
+        Name: //div[@class="box-cnt"]/div[@class="mar-t"]/a[not(@class)]/text()
+      Performers: &performers
+        Name: 
+          selector: //div[@class="name"]/span[@class="col-accent"]/text()
+          postProcess:
+          - replace: 
+            - regex: ( \(EU\))$
+              with: 
+      Image:
+        selector: //span[@id="spnPageUpdateTrailer"]/a/img/@data-src
+      Date: &date
+        selector: //div[@class="box-cnt"]/div[@class="mar-t"]/text()
```

Comment:
the date selector matches 2 divs (the second being the view/watch link) and the text a few more commas (you can have a look at the console with log set to debug when scraping)
The below selector seems to get us only the needed text
```yaml
        selector:  //div[@class="box-cnt"]/div[@class="mar-t"][not (contains(a, "the full"))]/text()[1]
```

Reply:
You suggestion works great